Add autodiscover clusters feature to cloudera integration (#13664)
DataDog/integrations-core: cloudera/README.md

<!-- xxz tab xxx -->
<!-- xxz tabs xxx -->

#### Clusters Discovery

> You can configure how your clusters are discovered with the `clusters` parameter.

Reviewer: ```suggestion
You can configure how your clusters are discovered with the `clusters` configuration option with the following parameters:
```

> `limit`

Reviewer: ```suggestion
- `limit`
```

: Maximum number of items to be autodiscovered.  
**Default value**: `None` (all clusters will be processed)

> `include`

Reviewer: ```suggestion
- `include`
```

: Mapping of regular expression keys and component config values to autodiscover.  
**Default value**: empty map

> `exclude`

Reviewer: ```suggestion
- `exclude`
```

: List of regular expressions with the patterns of components to exclude from autodiscovery.  
**Default value**: empty list

> 

Reviewer: I think you are also missing `interval` parameter here?

Author: Done

**Examples**:

> Process a maximum of `5` clusters with names starting with `my_cluster`:

Reviewer: ```suggestion
Process a maximum of `5` clusters with names that start with `my_cluster`:
```

```yaml
clusters:
  limit: 5
  include:
    - 'my_cluster.*'
```

> Process a maximum of `20` clusters and exclude those beginning with `tmp_`:

Reviewer: ```suggestion
Process a maximum of `20` clusters and exclude those with names that start with with `tmp_`:
```

<!-- xxz tab xxx -->
<!-- xxz tabs xxx -->

#### Clusters Discovery

You can configure how your clusters are discovered with the `clusters` configuration option with the following parameters:

- `limit`
: Maximum number of items to be autodiscovered.  
**Default value**: `None` (all clusters will be processed)

- `include`
: Mapping of regular expression keys and component config values to autodiscover.  
**Default value**: empty map

- `exclude`
: List of regular expressions with the patterns of components to exclude from autodiscovery.  
**Default value**: empty list

> - `limit`

Reviewer: ```suggestion
- `interval`
```
